sources/packages/backend/apps/db-migrations/src/sql/EducationProgramsOfferings/Rollback-update_offering-duplication-unique-index.sql

```diff
@@ -0,0 +1,60 @@
+DROP INDEX sims.loc_id_prog_id_offer_name_study_dts_year_study_index;
+
+-- Ensures currently duplicated offerings will be renamed to avoid issues
+-- while creating the unique constraint.
+UPDATE
+    sims.education_programs_offerings
+SET
+    offering_name = offering_name || ' - updated to be unique - ' || id :: text
+WHERE
+    id IN (
+        SELECT
+            offerings.id
+        FROM
+            sims.education_programs_offerings offerings
+            INNER JOIN (
+                SELECT
```

Comment:
@JasonCTang can we have this `SELECT` (**the SELECT part only**) query executed on PROD to have a better feeling if it would be updating any PROD record?